src/api/app/views/webui/staging/projects/_broken_packages.html.haml

```diff
@@ -7,6 +7,7 @@
   - if problems.empty?
     None
   - else
+    - build_states_not_built = ['unresolvable', 'broken']
```

Comment:
Maybe `failing_build_states`? Would it be better?

Reply:
Changed.

Reply:
Actually, thinking about this more, those are states without logs. `failed` is also a failing build state, but it does have logs, so it's not in this array. Sorry for the back and forth!

Reply:
Yes, better. Changed.